nixos/modules/services/games/asf.nix

```diff
@@ -0,0 +1,200 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.archisteamfarm;
+  asf-config = pkgs.writeText "ASF.json" (builtins.toJSON (cfg.config // {
+    # we disable it because ASF cannot update itself anyways
+    # and nixos takes care of restarting the service
+    # is in theory not needed as this is already the default for default builds
+    UpdateChannel = 0;
+  }));
```

Comment:
```suggestion
  format = pkgs.formats.json {};
  asf-config = format.generate "ASF.json" (cfg.config // {
    # we disable it because ASF cannot update itself anyways
    # and nixos takes care of restarting the service
    # is in theory not needed as this is already the default for default builds
    UpdateChannel = 0;
  });
```